app/controllers/subsidy/applications/edit.js

```diff
@@ -28,6 +37,95 @@ export default class SubsidyApplicationsEditController extends Controller {
   get canDelete() {
     return this.model.consumption.get('status.isConcept');
   }
+  @action
+  async exportSubsidyAsPDF() {
+    const previousDocumentTitle = document.title;
+    const filename = `${await this.createFilename()}.pdf`;
+
+    document.title = filename;
```

Comment:
Why is this needed exactly? What does the browser do with this? It might be worth adding a comment about that here. It feels strange without context.

Reply:
You're right, will add a comment. It's a trick to use the subsidy and step name in the pdf.